examples/TypeScriptMessaging/package.json

```diff
@@ -18,6 +18,8 @@
     "@react-navigation/native": "5.9.4",
     "@react-navigation/stack": "5.14.5",
     "@stream-io/flat-list-mvcp": "0.10.1",
+    "eslint-plugin-babel": "^5.3.1",
+    "eslint-plugin-markdown": "^2.2.1",
```

Comment:
Why is this installed here? And why as a `dependency` and not `devDependency`?

Reply:
I mistakenly added this and some other dependencies (which I removed). Removing these now